fun/connect4System/reactionListener.go.tmpl

```diff
@@ -0,0 +1,194 @@
+{{/* 
+    This is the reaction listener for the connect4 system
+
+    Trigger Type: Reaction
+    Trigger: Added Reactions Only
+    
+    Credits: 
+    zen | ゼン (https://github.com/z3nn13)
+*/}}
+
+{{ define "board_maker" }}
+        {{- $store := .store -}}
+        {{- $board := cslice.AppendSlice .board -}}
+        {{- $out := "" -}}
+        {{- range $board -}}
+                {{$rows := ""}}
+                {{- range . -}}
+                        {{- $rows = printf "%s%s " $rows (str . | $store.emojis.Get) -}}
+                {{- end -}}
+                {{- $out = print $rows "\n" $out -}}
+        {{- end -}}
+        {{- $out = print $out "1️⃣ 2️⃣ 3️⃣ 4️⃣ 5️⃣ 6️⃣ 7️⃣" -}}
+        {{- $embed := sdict "author" (sdict "name" (print .cPlayer.Username "'s turn") "icon_url" (.cPlayer.AvatarURL "256")) 
+            "title" "Connect4" 
+            "description" $out
+            "color" (str .turn | $store.color.Get)
+            "fields" (cslice 1 2)
+            "footer" (sdict "text" "Powered by • Yagpdb.xyz" ) -}}
+        {{- $embed.fields.Set (sub .turn 1) (sdict "name" (print "Player " .turn) "value" (print "> " .cPlayer.Mention) "inline" true) -}}
+        {{- $embed.fields.Set (sub .nextTurn 1 ) (sdict "name" (print "Player " .nextTurn) "value" .nPlayer.Mention "inline" true ) -}}
+        {{- .Set "embed" $embed -}}
+{{- end}}
+
+{{/* Global variables */}}
+{{ $data := sdict}}{{$players := cslice}}{{$cPlayer := ""}}{{$turn := ""}}{{$nextTurn := ""}}{{$nPlayer := ""}}{{$input := ""}}{{$flag := false}}
+{{ $store := sdict
+    "emojis" (sdict "0" "⚫" "1" "🔴" "2" "🟡" "10" "🔵") 
+    "color" (sdict "1" 0xff4d12 "2" 0xfff457)}}
+
+{{ if $db := dbGet 2021 "connect4" }}
+    {{ $data = sdict $db.Value}}
+    {{ $players = $players.AppendSlice $data.players}}{{/* slice ["offset",p1,p2] */}}
```

Comment:
```suggestion
    {{ $players = $data.players}}{{/* slice ["offset",p1,p2] */}}
```
DO NOT accept this suggestion (there's a bug with `in` and slices from DB currently) but just wanted to note this in case you changed it yourself based on my previous suggestion